### DISCUSSION TITLE
Bardaklar sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#296] [Zeka Sorusu]
Elimizde 24 birim, 11 birim ve 5 birim su alan üç bardak var.
24 birimlik bardak su ile dolu, diğerleri boş.
Yalnız bu bardaklar kullanılarak 12 birimlik su nasıl elde edilir?

**Comment by ArvenAA** [comment]
24 birimlik bardaktaki suyu 11 birimlik bardağa alıp oradan da 5 birimlik bardağa boşaltırız, 5 birimlik bardaktaki 5 birimlik suyu da herhangi bir kovaya boşaltırız. Sonrasında aynı işlemi tekrarlarız yani, 11 birimlik bardakta kalan 6 birimlik suyu döküp tekrar 24 birimlik bardakta kalan 13 birimlik suyu 11 birimlik bardağa boşaltırız, 11 birimlik bardaktaki suyu tekrar 5 birimlik bardağa boşaltıp oradan tekrar kovaya boşaltırız ve 11 birimlik bardakta kalan 6 birimlik suyu gene dökeriz. Bu durumda kovada 10 birim su, 24 birimlik bardakta ise 2 birimlik su kalır, 24 birimlik bardaktaki 2 birimlik suyu da içinde 10 birim su bulunan kovaya boşaltırız ve sonuç olarak kovada 12 birimlik su olur.

**Reply by sonsuzus** [reply]
Kova yok:) sadece bardaklar var.

**Comment by Alperg123456** [comment]
```
24     0     0
13     11     0
13     6     5
18     6     0
18     1     5
23     1     0
23     0     1
12     11     1
```

**Reply by sonsuzus** [reply]
Yazılmamış sorulara yönelelim.

**Reply by Alperg123456** [reply]
hocam ben diğer cevabı doğru kabul etmediğiniz için yaptım bu soruyu
